corehq/apps/users/device_rate_limiter.py

```diff
@@ -59,14 +59,9 @@ def rate_limit_device(self, domain, user_id, device_id):
         if device_count < self.device_limit_per_user(domain):
             self._track_usage(key, device_id)
             # this intentionally doesn't capture users with 1 device, only those with multiple
-            metrics_histogram(
-                'commcare.devices_per_user.device_count',
-                device_count + 1,
-                bucket_tag='count',
-                buckets=[3, 6, 11, 21, 51],
-                tags={'domain': domain, 'user_id': user_id},
+            metrics_counter(
+                'commcare.devices_per_user.additional_device', tags={'domain': domain, 'user_id': user_id}
```

Comment:
My understanding is that this will record a separate metric for each `(domain, user_id)` pair. Do you expect the number of pairs to be relatively low since users with a single device are excluded? How many unique pairs to we expect to see?

Reply:
Yeah fair question. We are billed by unique custom metrics per hour. Based on some past queries, it looked like we had close to 3000 users using multiple devices in a day (over estimate). If we average that out per hour, that is 125 per hour. There is an additional variance since the `host` tag could change for the same `user_id`. So I'd maybe anticipate a bump of 200 custom metrics per hour from this metric, which is substantial, but within our budget (for now). I think in order to get an idea of actual numbers, we should start here.